data/pokemon_mail.s

```diff
@@ -4,120 +4,120 @@
-    .byte 0xb4, 0xf3, 0x0e, 0x08
-    .byte 0x9c, 0xf3, 0x0e, 0x08
-    .byte 0x54, 0xf2, 0x0e, 0x08
-    .byte 0x34, 0xf2, 0x0e, 0x08
-    .byte 0x10, 0xf1, 0x0e, 0x08
-    .byte 0xf0, 0xf0, 0x0e, 0x08
-    .byte 0x04, 0xf0, 0x0e, 0x08
-    .byte 0xe4, 0xef, 0x0e, 0x08
-    .byte 0xb8, 0xee, 0x0e, 0x08
-    .byte 0xa0, 0xee, 0x0e, 0x08
-    .byte 0x68, 0xed, 0x0e, 0x08
-    .byte 0x48, 0xed, 0x0e, 0x08
-    .byte 0x3c, 0xec, 0x0e, 0x08
-    .byte 0x24, 0xec, 0x0e, 0x08
-    .byte 0xd8, 0xea, 0x0e, 0x08
-    .byte 0xc0, 0xea, 0x0e, 0x08
-    .byte 0x84, 0xe9, 0x0e, 0x08
-    .byte 0x68, 0xe9, 0x0e, 0x08
-    .byte 0x74, 0xe8, 0x0e, 0x08
-    .byte 0x58, 0xe8, 0x0e, 0x08
-    .byte 0x20, 0xe7, 0x0e, 0x08
-    .byte 0x04, 0xe7, 0x0e, 0x08
-    .byte 0xcc, 0xe5, 0x0e, 0x08
-    .byte 0xb0, 0xe5, 0x0e, 0x08
-    .byte 0x78, 0xe4, 0x0e, 0x08
-    .byte 0x58, 0xe4, 0x0e, 0x08
-    .byte 0x1c, 0xe3, 0x0e, 0x08
-    .byte 0x04, 0xe3, 0x0e, 0x08
-    .byte 0xc8, 0xe1, 0x0e, 0x08
-    .byte 0xc0, 0xe1, 0x0e, 0x08
-    .byte 0x80, 0xe0, 0x0e, 0x08
-    .byte 0x68, 0xe0, 0x0e, 0x08
-    .byte 0x0c, 0xdf, 0x0e, 0x08
-    .byte 0xf8, 0xde, 0x0e, 0x08
-    .byte 0x2c, 0xde, 0x0e, 0x08
-    .byte 0x20, 0xde, 0x0e, 0x08
-    .byte 0xd4, 0xdc, 0x0e, 0x08
-    .byte 0xb4, 0xdc, 0x0e, 0x08
-    .byte 0x84, 0xdb, 0x0e, 0x08
-    .byte 0x68, 0xdb, 0x0e, 0x08
-    .byte 0x40, 0xda, 0x0e, 0x08
-    .byte 0x1c, 0xda, 0x0e, 0x08
-    .byte 0xf0, 0xd8, 0x0e, 0x08
-    .byte 0xd0, 0xd8, 0x0e, 0x08
-    .byte 0xf8, 0xd7, 0x0e, 0x08
-    .byte 0xdc, 0xd7, 0x0e, 0x08
-    .byte 0x00, 0xd7, 0x0e, 0x08
-    .byte 0xe8, 0xd6, 0x0e, 0x08
-    .byte 0xac, 0xd5, 0x0e, 0x08
-    .byte 0x94, 0xd5, 0x0e, 0x08
-    .byte 0x78, 0xd4, 0x0e, 0x08
-    .byte 0x5c, 0xd4, 0x0e, 0x08
-    .byte 0x8c, 0xd3, 0x0e, 0x08
-    .byte 0x70, 0xd3, 0x0e, 0x08
-    .byte 0xa8, 0xd2, 0x0e, 0x08
-    .byte 0x90, 0xd2, 0x0e, 0x08
-    .byte 0x60, 0xd1, 0x0e, 0x08
-    .byte 0x40, 0xd1, 0x0e, 0x08
-    .byte 0x78, 0xd0, 0x0e, 0x08
-    .byte 0x54, 0xd0, 0x0e, 0x08
-    .byte 0x64, 0xcf, 0x0e, 0x08
-    .byte 0x44, 0xcf, 0x0e, 0x08
-    .byte 0xf8, 0xcd, 0x0e, 0x08
-    .byte 0xd8, 0xcd, 0x0e, 0x08
-    .byte 0xb0, 0xcc, 0x0e, 0x08
-    .byte 0x94, 0xcc, 0x0e, 0x08
-    .byte 0x8c, 0xcb, 0x0e, 0x08
-    .byte 0x70, 0xcb, 0x0e, 0x08
-    .byte 0x68, 0xca, 0x0e, 0x08
-    .byte 0x44, 0xca, 0x0e, 0x08
-    .byte 0x40, 0xc9, 0x0e, 0x08
-    .byte 0x24, 0xc9, 0x0e, 0x08
-    .byte 0xec, 0xc7, 0x0e, 0x08
-    .byte 0xd4, 0xc7, 0x0e, 0x08
-    .byte 0xa0, 0xc6, 0x0e, 0x08
-    .byte 0x7c, 0xc6, 0x0e, 0x08
-    .byte 0x40, 0xc5, 0x0e, 0x08
-    .byte 0x20, 0xc5, 0x0e, 0x08
-    .byte 0x10, 0xc4, 0x0e, 0x08
-    .byte 0xf0, 0xc3, 0x0e, 0x08
-    .byte 0xf4, 0xc2, 0x0e, 0x08
-    .byte 0xdc, 0xc2, 0x0e, 0x08
-    .byte 0xd8, 0xc1, 0x0e, 0x08
-    .byte 0xc0, 0xc1, 0x0e, 0x08
-    .byte 0x9c, 0xc0, 0x0e, 0x08
-    .byte 0x84, 0xc0, 0x0e, 0x08
-    .byte 0x78, 0xbf, 0x0e, 0x08
+    .4byte WelcomeText
```

Comment:
Getting linker errors because of duplicate names.. Maybe have a prefix of `gMail_****` to fix that up